i18n/vi.json

```diff
@@ -0,0 +1,282 @@
+{
+  "I18N_ABOUT_PAGE_TITLE": "Về Chúng Tôi - Oppia",
+  "I18N_ACTION_BROWSE_EXPLORATIONS": "Tra cứu các cuộc hành trình",
+  "I18N_ACTION_CREATE_EXPLORATION": "Tạo cuộc hành trình của bạn",
+  "I18N_CREATE_ACTIVITY_QUESTION": "Bạn muốn tạo ra cái gì?",
+  "I18N_CREATE_ACTIVITY_TITLE": "Tạo ra hoạt động mới",
+  "I18N_CREATE_COLLECTION": "Tạo nhóm",
+  "I18N_CREATE_EXPLORATION": "Tạo cuộc hành trình",
+  "I18N_CREATE_EXPLORATION_CREATE": "Sáng Tạo",
```

Comment:
feels kinda unnatural, maybe we should find its usage somewhere in Oppia in order to have a smoother translate?